doc/firstWeek summary.md

```diff
@@ -0,0 +1,25 @@
+## First week summary with question and answers.
+
+### Question- Why OAM users needs GItOps? 
+
+> OAM users needs GitOps because of multiple reasons some of them listed below -
+- <b>Deploy Faster More Often</b> When we say “high velocity” we mean that every product team can safely ship updates many times a day — deploy instantly, observe the results in real time, and use this feedback to roll forward or back.
+- <b>Easy and Fast Error Recovery</b> — The Git record is then not just an audit log but also a transaction log. You can roll back & forth to any snapshot.
+GitOps you have a complete history of how your environment changed over time. This makes error recovery as easy as issuing a git revert and watching your environment being restored.
+
+- <b>Easier Credential Management</b> — GitOps allows you to manage deployments completely from inside your environment. For that, your environment only needs access to your repository and image registry. That’s it. You don’t have to give your developers direct access to the environment.
+- <b>Modular architecture</b> -: GitOps has three parts: the Git repo, the deployment process, and possibly a process that automates version updates in the Git repo. All three of these can evolve or be replaced independently of each other.
+- <b>Tool-independent architecture</b> -:The modularity mentioned in point No. 4 directly leads to the fact that a GitOps architecture is one where you can assemble the best tools in a flexible way. Any popular Git server will do the Git part of course, and Flux or ArgoCD can take care of syncing the repo to the cluster. JenkinsX is a tool which handles all parts of this process, including creating the Git repos and updating them with new version updates when a new Docker image is built.
+
+### Question- What the additional values provide by Argo or Flux? Why we choose one over other?
+
+We believe both projects, Flux and ArgoCD have very interesting and powerful capabilities to manage deployments in Kubernetes, but each come with pros and cons that should be evaluated for our specific case. These tools aren’t really competing with each other, rather they are aiming to fulfill different use-cases.<br>
```

Comment:
I think this is good enough for first week report.

For the following week, the task is to try out Argo and Flux on Kubernetes. Answer the following questions:

- What are the advantages and disadvantages of Argo and Flux? In which case we would choose one over the other?
- If I have an [OAM app](https://github.com/crossplane/oam-kubernetes-runtime/tree/master/examples/flight-tracker), how can I integrate that with these tools?